.github/workflows/cmake.yml

```diff
@@ -54,7 +55,7 @@ jobs:
             cxx: "g++",
           }
           - {
-            os: ubuntu-20.04,
+            os: ubuntu-22.04,
```

Comment:
With this version I have `ctest` errors like on OSX!

Reply:
Yes I see that, I'm also repeating it in a u22 vm,  Interesting it works ok in fedora 37.
I suspect something has changed in the lastest version of zeromq, it probably reporting new errors